app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :author
```

Comment:
- [ ] Please specify the `class_name` when setting up the association between a `like` and an `author`
- [ ]  According to requirements, your `Like` model should have a method implemented that updates the likes counter for a post.